packages/runtime/src/events/handleAction.ts

```diff
@@ -244,9 +244,11 @@ export function handleAction(
               subEvent.actions.forEach((action) =>
                 handleAction(
                   action,
-                  eventData
```

Comment:
Could we perhaps introduce/modify a linter rule to catch this going forward? It would be safer if we could only compare on booleans?